src/components/2019/year-in-review/Multimedia.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import s from 'styled-components'
+import { PhotoGallery, SectionHeader, SectionMainText, MainHeader } from './Typograph'
+
+const Spacer = s.div`
+    margin-bottom: 50px;
+`
+
+const Wrapper = s.div`
+    background-color: #131313;
+    padding: 120px 0px;
+`
+
+const MultiMedia = () => {
+  return (
+        <Wrapper id="multimedia">
+            <div class="container">
+                <SectionHeader color={"#ffffff"}>Multimedia</SectionHeader>
+                <div>
+                    <a href="https://www.thedp.com/article/2019/12/penn-decade-in-review-2010s-highlights-photo-essay-gallery">
+                        <PhotoGallery className="img-fluid" src="https://snworksceo.imgix.net/dpn/8baab047-09cd-40cf-9033-87fc736f07b8.original.jpg" />
+                        <MainHeader color={"white"}>Photo Essay | Penn’s standout moments from the 2010s</MainHeader>
+                    </a>
+                    <SectionMainText color={"#f7f7f7"}>The 2010s are coming to a close, and a lot has happened at Penn in the last decade. It was a decade of firsts: the first Penn alumnus to be to elected President of the United States, the first Philadelphia Eagles Super Bowl victory, and the first Women's March. There were several record-breaking and newsworthy athletic achievements. Important figures such as Pope Francis and LeBron James visited the University. And Penn's campus continued to expand with new construction projects.
+                            Take a look back at the past decade at Penn in photos.</SectionMainText>
+                </div>
+                <Spacer/>
+                <div>
+                    <div class="embed-container d-flex justify-content-center">
+                        <iframe width="560" height="315" src="https://www.youtube.com/embed/FMH7sXPXf7c" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe> </div>
+                     <div>
+                </div>
+                <MainHeader color={"white"} className="mt-4">The Story of 2019</MainHeader>
```

Comment:
when using a styled component, I would prefer styles to be consolidated in the styling for the styled component. Please move the appropriate styles for "mt-4" into the styled-component css

Reply:
this doesn't apply for more complex bulma classes, but for something like mt-4 which applies margin top and only that, please do simplify the html